nyt-top-20-science-news/README.md

```diff
@@ -0,0 +1,83 @@
+# NYT: Top 20 Science News
+This website shows you the **20** most popular New York Times science news stories.
+## Requirements:
+This solution aims to fulfill the following requirements, in order of priority:
+- Display a list of articles.
+- Implement a search feature to dynamically filter articles.
+- Search terms should match against the following article attributes:
+    - title
+    - byline
+    - section
+
+## Getting started
+Before you start, please make sure to set up the correct environment variable. At the root of folder (nyt-top-20-sience-news), please create a `.env` file with the following content:
+
+```
+REACT_APP_NYT_API_KEY=<YOUR API KEY HERE>
+```
+You can find an API key in the parent README.md.
+
+Visit the website [here](https://kevb10.github.io/code-challenge/index.html).
+
+- `yarn install`: Installs dependencies for the project.
+- `yarn start`: Runs app in development mode.
+- `yarn test`: Runs the test watcher in an interactive mode.
+- `yarn run deploy`: Builds and deploys app to github pages.
+- `yarn lint`: Run linter. Use `--fix` to fix all fixable issues.
+
+
+## Process
```

Comment:
This page isn't at all required, but it is very appreciated. And this section specifically is so helpful in understanding how you approached this project. 👍